File: JavaBasic/90_blood_donate.java
// 90	90_blood_donate.java	Determine whether a person can donate blood based on age and weight.
public class _90_blood_donate {
    public static void main(String[] args) {
        int age = 22; double weight = 55;
        System.out.println("Age: " + age + ", Weight: " + weight + " kg");
        if (age >= 18 && age <= 65 && weight >= 50) System.out.println("Eligible to donate blood.");
        else System.out.println("NOT eligible to donate blood.");
    }
}

File: JSBASIC/90_blood_donate.js
// 90	90_blood_donate.js	Determine whether a person can donate blood based on age and weight.

let age = 22;
let weight = 55; // kg
console.log("Age:", age, "| Weight:", weight, "kg");
if (age >= 18 && age <= 65 && weight >= 50) {
    console.log("Eligible to donate blood.");
} else {
    console.log("NOT eligible to donate blood.");
}
